Prevention Act;Prevents sandmining, scamming and trapping.html

```diff
@@ -0,0 +1,18 @@
+<h2>Prevention</h2>
+<h3>Sandmining</h3>
+<p>
+    Do not sand mine, if a player does sand mine the military may be deployed
```

Comment:
`Do not sand mine` sounds as if you are talking to someone, laws are not talking to anyone.
Also it's `sandmine` not `sand mine`.